OpenUpSA/wazimap-ng
# Added option for adding boolean for linear scrubber in profile indicator admin/api

## Description
Added config to enable linear scrubber in Mapchip

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-407

## How to test it locally
* Go to profile indicator
* check option enable_linear scrubber
* you should get warning after checking the checkbox .. if unchecked warning message should hide

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
